Products/ZenModel/fixZauthHealthCheck.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import os
+import re
+import logging
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+sm.require("1.0.0")
+
+
+class FixZauthHealthCheck(Migrate.Step):
+    "Use different curl request to prevent `authentication failed` spam in audit.log"
```

Comment:
FYI, the idiomatic way to write documentation strings in Python is to use triple quotes, i.e.
```
    """Use different curl request to prevent 'authentication failed' span in audit.log"""
```